guides/micronaut-oracle-autonomous-db/micronaut-oracle-autonomous-db.adoc

```diff
@@ -261,36 +259,45 @@ You can run some cURL requests to test the application:
 
 [source,bash]
 ----
-complete $ curl -i localhost:8080/things
+curl -i localhost:8080/things
+----
+
+```
 HTTP/1.1 200 OK
 Content-Type: application/json
 content-length: 49
 connection: keep-alive
 
 [{"id":1,"name":"Fred"},{"id":2,"name":"Barney"}]
-----
+```
 
 [source,bash]
 ----
-complete $ curl -i localhost:8080/things/Fred
+curl -i localhost:8080/things/Fred
+----
+
+```
 HTTP/1.1 200 OK
 Content-Type: application/json
 content-length: 22
 connection: keep-alive
 
 {"id":1,"name":"Fred"}
-----
+```
 
 [source,bash]
 ----
-complete $ curl -i localhost:8080/things/Ralph
+curl -i localhost:8080/things/Ralph
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks